spec/system/work_wizard_spec.rb

```diff
@@ -83,6 +83,8 @@
 
     click_on "Next"
     expect(page).to have_css(validate_form_css)
+    # Force the work to have two files (readme + another file)
+    stub_s3 data: [FactoryBot.build(:s3_readme, work:), FactoryBot.build(:s3_file, work:)]
```

Comment:
This seems to be the culprit. We are not adding a data file (other than the readme) and the code expects one. That's why the validation error in CI is actually indicating. 

What I still don't understand is how come this passed sometimes but not others?